netcdf/src/attribute.rs

```diff
@@ -104,6 +104,7 @@ impl<'a> Attribute<'a> {
                     Ok(AttrValue::Uchar(value))
                 }
                 len => {
+                    #[allow(clippy::unnecessary_cast)]
```

Comment:
Is it worth keeping the cast here and in `File::path`?

Reply:
This one is indeed a NOP, but the one in `File::path` changes from `size_t` to `usize` which could be different

Reply:
Right, clippy flags it because of https://github.com/georust/netcdf/blob/master/netcdf-sys/src/functions.rs#L52.

Reply:
Here are the docs for `size_t`: https://doc.rust-lang.org/core/ffi/type.c_size_t.html. I will remove the cast, thanks!